torch/ao/quantization/quantize_fx.py

```diff
@@ -632,6 +632,22 @@ def convert_fx(
                  }
               },
 
+              # optional: specify the path for standalone modules
```

Comment:
it's unexpected that both `prepare_fx` and `convert_fx` support different flavors of standalone module, can we converge on just one and only have it in one place?

Reply:
yeah ideally we should do that. currently the module type is not preserved in symbolic tracing, so we need to add that first, I remember we have discussed about this at some point, not sure what is the plan on that, do you remember what is the plan for preserving the types in symbolic tracing?

Reply:
can the new functionality live in the same places as the old one (either both live in prepare, or both live in convert), and can we add a comment on the restrictions and plans to remove it?

Reply:
I just looked at the code again, and I find that we can remove this since we are supporting the backend_config_dict only in the new convert flow, this config was added initially because I was planning to support this in the current convert function. will remove this part of docs